examples/sha256/table16/compression/compression_util.rs

```diff
@@ -278,66 +325,68 @@ impl Compression {
         Ok(RoundWordE::new(e_pieces, dense_halves, spread_halves))
     }
 
-    pub(super) fn assign_upper_sigma_0<F: FieldExt>(
-        &self,
-        region: &mut Region<'_, Table16Chip<F>>,
+    pub(super) fn assign_upper_sigma_0(
+        &mut self,
         idx: i32,
         word: AbcdVar,
     ) -> Result<(CellValue16, CellValue16), Error> {
+        let configured = self.configured().clone();
```

Comment:
```suggestion
        let configured = self.configured().clone();
```
There must be a way to avoid these `clone()`s, I think.